src/core/index.mld

```diff
@@ -15,30 +15,28 @@ Here is a simplistic Lwt program which requests the Google front page, and fails
 if the request is not completed in five seconds:
 
 {[
-open Lwt.Syntax
-
 let () =
   let request =
-    let* addresses = Lwt_unix.getaddrinfo "google.com" "80" [] in
+    let%lwt addresses = Lwt_unix.getaddrinfo "google.com" "80" [] in
     let google = Lwt_unix.((List.hd addresses).ai_addr) in
 
     Lwt_io.(with_connection google (fun (incoming, outgoing) ->
-      let* () = write outgoing "GET / HTTP/1.1\r\n" in
-      let* () = write outgoing "Connection: close\r\n\r\n" in
-      let* response = read incoming in
+      write outgoing "GET / HTTP/1.1\r\n";%lwt
+      write outgoing "Connection: close\r\n\r\n";%lwt
+      let%lwt response = read incoming in
       Lwt.return (Some response)))
   in
 
   let timeout =
-    let* () = Lwt_unix.sleep 5. in
-    Lwt.return None
+    Lwt_unix.sleep 5.;%lwt
+    Lwt.return_none
   in
 
   match Lwt_main.run (Lwt.pick [request; timeout]) with
   | Some response -> print_string response
   | None -> prerr_endline "Request timed out"; exit 1
 
-(* ocamlfind opt -package lwt.unix -linkpkg example.ml && ./a.out *)
+(* ocamlfind opt -package lwt.unix,lwt_ppx -linkpkg example.ml && ./a.out *)
 ]}
 
 In the program, functions such as [Lwt_io.write] create promises. The
```

Comment:
Also note that some of these examples actually were using the PPX originally, as you can see in the next line: